website/content/v1.8/kubernetes-guides/network/deploying-cilium.md

```diff
@@ -359,14 +359,24 @@ We can leverage this to our advantage to install Cilium by using an inline manif
                 command:
                   - cilium
                   - install
-                  - --set ipam.mode=kubernetes
-                  - --set kubeProxyReplacement=true
-                  - --set securityContext.capabilities.ciliumAgent="{CHOWN,KILL,NET_ADMIN,NET_RAW,IPC_LOCK,SYS_ADMIN,SYS_RESOURCE,DAC_OVERRIDE,FOWNER,SETGID,SETUID}"
-                  - --set securityContext.capabilities.cleanCiliumState="{NET_ADMIN,SYS_ADMIN,SYS_RESOURCE}"
-                  - --set cgroup.autoMount.enabled=false
-                  - --set cgroup.hostRoot=/sys/fs/cgroup
-                  - --set k8sServiceHost=localhost
-                  - --set k8sServicePort=7445
+                  - --set
+                  - ipam.mode=kubernetes
+                  - --set
+                  - kubeProxyReplacement=true
+                  - --set
+                  - securityContext.privileged=true
```

Comment:
I'm not sure about this, we don't seem to be setting this anywhere, and not used in the tests.

The formatting of arguments was certainly wrong, and thanks for fixing